core/src/main/java/com/netflix/conductor/core/utils/JsonUtils.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.conductor.core.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Map;
+
+@Singleton
```

Comment:
Why define it as a `@Singleton`  when this util class does not have any dependency and while the initialization in all the classes is done by calling the constructor `private final JsonUtils jsonUtils = new JsonUtils();` ?